public/javascripts/SVLabel/src/SVLabel/task/Task.js

```diff
@@ -52,6 +73,26 @@ function Task (geojson, tutorialTask, currentLat, currentLng, startPointReversed
         }
 
         paths = null;
+
+        angle = null;
```

Comment:
it looks like you could initialize a lot of these where you declare the variables, and might as well do it if don't require anything special. at the very least, initializing things to null and empty lists is fine to do up above.